packages/manager/sms/src/users/change-password/telecom-sms-users-change-password.html

```diff
@@ -1,101 +1,71 @@
-<form name="changePasswordUserForm"
-      id="changePasswordUserForm"
-      data-ng-submit="UsersChangePasswordCtrl.changePassword()">
+<form name="$ctrl.changePasswordUserForm"
+      id="$ctrl.changePasswordUserForm"
+      data-ng-submit="$ctrl.changePassword()">
 
     <div class="modal-header">
         <button
             type="button"
             class="close float-right"
-            aria-label="{{ ::'sms_cancel' | translate }}"
-            data-ng-click="UsersChangePasswordCtrl.cancel()">
+            aria-label="{{ ::'cancel' | translate }}"
```

Comment:
There is no reason to specify previous translation key I guess. You can stay with all news.